app/Services/UserService.php

```diff
@@ -48,4 +49,13 @@ public function updatePassword($checkToken, $password): array
 
         return ['message' => 'Outdated token'];
     }
+    public function updateUser($id, $name, $email)
```

Comment:
as I said before: $id, array $data . Do not forget to add type hint input and output values